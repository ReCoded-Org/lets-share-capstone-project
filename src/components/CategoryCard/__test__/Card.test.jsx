"use strict";

import renderer from "react-test-renderer";

import Card from "../card.jsx";

it("renders correctly", () => {
    const tree = renderer.create(<Card />).toJSON();
    expect(tree).toMatchSnapshot();
});
