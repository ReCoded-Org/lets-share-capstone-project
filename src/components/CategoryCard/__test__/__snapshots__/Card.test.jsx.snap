// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="group mx-auto flex h-fit w-fit flex-col items-center justify-between rounded-lg p-4 text-center shadow-xl hover:animate-bounce "
>
  <svg
    className="h-6 w-6 "
    fill="#F07167"
    stroke="currentColor"
    strokeWidth={2}
    viewBox="0 0 24 24"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M7 7h.01M7 3h5c.512 0 1.024.195 1.414.586l7 7a2 2 0 010 2.828l-7 7a2 2 0 01-2.828 0l-7-7A1.994 1.994 0 013 12V7a4 4 0 014-4z"
      strokeLinecap="round"
      strokeLinejoin="round"
    />
  </svg>
  <span
    className="mt-4 font-bold text-primary"
  >
    Category
  </span>
</div>
`;
